client/packages/system/src/Vaccination/Components/VaccineCardTable.tsx

```diff
@@ -40,11 +40,11 @@ const isPreviousDoseGiven = (
   return itemsForCourse[doseIndex - 1]?.given;
 };
 
-const isRowClickable = (
-  isEncounter: boolean,
+const includeRow = (
```

Comment:
honestly I don't know what to name this, but needed to be more generic, because in Patient view, the "next" doses should NOT be clickable, but we do want to show their status